### DISCUSSION TITLE
Midi/OSC controls

### DISCUSSION THREAD

**Discussion by mapeje** [mawoka-myblock/ClassQuiz#312] [Ideas]
Our band sometimes use Kahoot/Menti to create a live music quiz. It would be fantastic if we could start using ClassQuiz instead! One amazing feature would be to somehow control the quiz from midi/osc. This way, we could sync the questions to our music setlist. Or we could use a midi controller to easier control the game.

**Comment by mawoka-myblock** [comment]
Unfortunately, I don't own any device to test that with, so either one who owns such a device will have to add this feature, or I somehow get a MIDI controller

**Reply by mapeje** [reply]
I think if it could be possible to add kyboard shortcuts like Return, arrovs or Space to advance in the quiz, there are midi-to-keyboard solutions that I am sure would work.
Would it be a difficult taks to add keyboard shortcuts?

**Reply by mawoka-myblock** [reply]
Keyboard shortcuts are easily possible. Will look into that. https://github.com/mawoka-myblock/ClassQuiz/issues/314

**Reply by mapeje** [reply]
Amazing! Thanks!!
